xdp-dump/xdpdump.h

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: GPL-2.0
+
+/******************************************************************************
+ * Multiple include protection
+ ******************************************************************************/
+#ifndef _XDPDUMP_H_
+#define _XDPDUMP_H_
+
+/******************************************************************************
+ * General definitions
+ ******************************************************************************/
+#define PERF_MMAP_PAGE_COUNT	16
+#define MAX_CPUS		256
+
+/******************************************************************************
+ * General used macros
+ ******************************************************************************/
+#ifndef offsetof
+#define offsetof(type, member) ((size_t) &((type *)0)->member)
+#endif
+
+#ifndef container_of
+#define container_of(ptr, type, member) ({                    \
+	const typeof( ((type *)0)->member ) *__mptr = (ptr);  \
+	(type *)( (char *)__mptr - offsetof(type,member) );})
+#endif
+
+#ifndef min
+#define min(x, y) ((x) < (y) ? (x) : (y))
+#endif
```

Comment:
Maybe we should move all these into utils.h?

Reply:
Is see min() is already there, but I needed it in the bpf program (not anymore it looks like) so that's why it was here. The others I'll move there.

Reply:
Right, cool